src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml

```diff
@@ -71,7 +81,7 @@
         <script src="~/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.2.0.min.js"
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.3.1.min.js"

```

Comment:
I hope you ensured that Bootstrap version we use supports jQuery 3 ;)

Reply:
yes :)

Reply:
http://blog.getbootstrap.com/2016/07/25/bootstrap-3-3-7-released/